app/Services/Validation/Rules/TwitterHandleRule.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Services\Validation\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Illuminate\Support\Facades\Lang;
+
+class TwitterHandleRule implements Rule
+{
+    /**
+     * Validation for Twitter handlers.
+     *
+     * @var string
+     */
+    const PATTERN = '^[A-Za-z0-9_]{1,10}$';
```

Comment:
Doesn't this limit it to 10 characters? It should be 15. While you're at it, mention the length limit in the docstring below.